src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerCopyFileToContainer.groovy

```diff
@@ -41,7 +42,7 @@ class DockerCopyFileToContainer extends DockerExistingContainer {
      */
     @Input
     @Optional
-    Closure<File> tarFile
+    final RegularFileProperty tarFile = newInputFile()
```

Comment:
You should probably change the annotations to something like:
```
@InputFile
@PathSensitive(PathSensitivity.RELATIVE)
```

Reply:
Yes, absolutely in the long run. At the moment the task is not marked with `@Cacheable` so the path sensitivity wouldn't make much of a difference. I did this for `Dockerfile` which already is cacheable.